index.rst

```diff
@@ -46,77 +46,233 @@
+The epic description must contain the dates for which the epic is valid, which is set by the ``Start date`` and ``End date`` parameters. 
+Specifics to the run, such as filter changes or any major system changes, should be listed in the epic.
+Most of the coordination and system details regarding run coordination are contained in an associated Confluence page night log, this is discussed in a following section.
+Of course, all tasks (or Jira stories) that are planned for execution during this block are to be linked to the epic.
+
+
+
+Proposing an Observation Activity
+=================================
+
+There are numerous types of observing activities, however, they are often categorized into scientific campaigns (e.g. imaging or spectral surveys) and engineering tasks.
+
+- *Campaigns* are meant to indicate a broad set of observations to study system performance or take new science data.
+  They often consist of a survey of spectral and/or imaging data, and the visits are coordinated by the Scheduler.
+  Although not required, campaigns are generally multi-night studies and not one-off characterizations.
+- *Engineering Tasks* encompass activities that are meant to be close to single-occurrence observations.
+  This includes functionality testing and characterization studies (e.g. AOS LUT generation), but can also include discretized science observations that are limited in scope and duration.
+  These tasks are generally run in a more manual fashion, either via the execution of SAL scripts via the scriptQueue CSC, or when required, via a Jupyter notebook running off of Nublado.
+
+The proposal of an observing activity is done via a Jira ticketing process.
+
+Jira Ticket Creation
+--------------------
+
+The jira ticket for a given observation contains all the information to execute an observation, as well as a justification for the observation.
+It is acceptable to attach small files and/or link other documents where applicable, specifically in regards to the justification.
+The prioritization of the observation is handled by the commissioning leads, but all tasks must go through proper preparations to maximize on-sky efficiency. 
+This is discussed further in the `Observation Preparation`_ section.
+
+When proposing an observation, a jira ticket (story), in the `SITCOM Jira Project <https://jira.lsstcorp.org/projects/SITCOM/issues/SITCOM-310?filter=allopenissues>`_.
+Be sure to complete the following steps:
+
+#. Link the epic in which the proposal is to performed
+#. Add a brief description of the observation, and a justification.
+   Linking supporting documentation is encouraged.
+#. Include condition constraints, specifically seeing, transparency, and an evaluation of stability (if appropriate).
+#. Include details on any required supporting hardware (e.g. the DIMM must be operational)
+#. Set the reviewer to ``pingraham``, *and also them add as a watcher*. 
```

Comment:
*and also them add as a watcher*. ??

Are we expecting this to hold for MT as well? I think my question is, who do we expect to be the reviewer(s) in terms of title? Maybe add some comment about the role of the reviewer?

Reply:
So when added as a reviewer, you only get messages when the ticket is marked as In Review... but I want to see them when they're created/updated. 
I modified the sentence a bit.
Zooming out, this is a very weak point of the procedure that needs to be improved.

Reply:
I also added a sentence about the reviewer. Honestly, this won't hold for the MT... I think we need a new workflow to really accommodate the complexities here. Not to mention that always using me creates a single-point-of-failure.